<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Port</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>188</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="portLable">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="BaudRateLable">
         <property name="text">
          <string>Baud Rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dataBitsLable">
         <property name="text">
          <string>Data Bits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ParityLable">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stopBitsLable">
         <property name="text">
          <string>Stop Bits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="flowcontrolLable">
         <property name="text">
          <string>Flow Control:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="portNameBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="baudRateBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="dataBitBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="parityBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="stopBitBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="flowcontrlBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>421</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="dataToSendEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="echoSendCheckBox">
         <property name="text">
          <string>Echo send</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="rxConfigGroup">
       <property name="title">
        <string>Recieve:</string>
       </property>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>191</width>
           <height>24</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="rxAsciiChecked">
            <property name="text">
             <string>ASCII</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rxHexChecked">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Hex</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="txConfigGroup">
       <property name="title">
        <string>Transmit:</string>
       </property>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>191</width>
           <height>24</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="txAsciiChecked">
            <property name="text">
             <string>ASCII</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="txHexChecked">
            <property name="text">
             <string>Hex</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>textBrowser</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionSave"/>
   <addaction name="actionClear"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../../../../opt/qt/Qt5.6.0/Examples/Qt-5.6/qtserialport/serialport/terminal/terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../../../../opt/qt/Qt5.6.0/Examples/Qt-5.6/qtserialport/serialport/terminal/terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../../../../opt/qt/Qt5.6.0/Examples/Qt-5.6/qtserialport/serialport/terminal/terminal.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../../../opt/qt/Qt5.6.0/Examples/Qt-5.6/qtserialport/serialport/terminal/terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About Author</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
